src/Parser/Core/Modules/DamageTaken.js

```diff
@@ -82,29 +117,44 @@ class DamageTaken extends Analyzer {
         tooltip={
           `The total damage taken was ${formatThousands(this.total.effective)} (${formatThousands(this.total.overkill)} overkill).`
         }
-        footer={(
-          <div className="statistic-bar">
-            {Object.keys(this._byMagicSchool)
-              .map(type => {
-                const effective = this._byMagicSchool[type].effective;
-                return (
-                  <div
-                    key={type}
-                    className={`spell-school-${type}-bg`}
-                    style={{ width: `${effective / this.total.effective * 100}%` }}
-                    data-tip={
-                      `Damage taken by magic school:
-                      <ul>
-                        ${Object.keys(this._byMagicSchool)
-                          .map(type => `<li><b>${MAGIC_SCHOOLS.names[type] || 'Unknown'}</b>: ${formatThousands(this._byMagicSchool[type].effective)} (${formatPercentage(this._byMagicSchool[type].effective / this.total.effective)}%)</li>`)
-                          .join('')}
-                      </ul>`
-                    }
-                  />
-                );
-              })}
+        footer={
+            [(<div id="damage-taken-bar-detailed" onClick={() => this.handleClick()} className="statistic-bar" style={{display: "none"}}>
+                {Object.keys(this._byMagicSchool)
+                .map(type => {
+                  const effective = this._byMagicSchool[type].effective;
+                  if(effective === 0) return ``;
```

Comment:
I think it would be cleaner if this were a `.filter(..)` before the map. Example:

```jsx
Object.keys(this._byMagicSchool)
  .filter(type => this._byMagicSchool[type].effective !== 0)
  .map(type => {
    // etc
```